assembly/src/ast/imports.rs

```diff
@@ -0,0 +1,91 @@
+use super::{
+    BTreeMap, ByteReader, ByteWriter, Deserializable, DeserializationError, ImportedProcsMap,
+    LibraryPath, ProcedureId, ProcedureName, Serializable, String, ToString, Vec,
+};
+
+/// Information about imports stored in the AST
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ModuleImports {
+    /// Imported libraries
+    imports: BTreeMap<String, LibraryPath>,
+    /// Imported procedures that are called from somewhere in the AST
+    used_imported_procs: ImportedProcsMap,
+}
+
+impl ModuleImports {
```

Comment:
Since this struct is a part of the public interface, I'd add doc comments to most of its methods.

Reply:
Fixed.